src/app/not-found.tsx

```
@@ -0,0 +1,36 @@
+"use client";
+
+export default function NotFoundPage() {
+    return (
+        <div className="flex min-h-screen flex-col justify-between p-8 font-[family-name:var(--font-geist-sans)] sm:p-20">
+            <main className="flex flex-grow flex-col items-center justify-center text-center">
+                <h1 className="text-3xl">
+                    {"You're lost in the woods!"}
+                </h1>
+                {/* Add single horizontal line to the above heading */}
+                <div className="mt-4 h-px w-96 bg-gray-700/70 rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)